src/main/java/org/elasticsearch/index/mapper/core/AbstractFieldMapper.java

```diff
@@ -360,6 +368,14 @@ protected String defaultPostingFormat() {
     protected String defaultDocValuesFormat() {
         return null;
     }
+    
+    protected boolean defaultDocValues() {
+        if (writePre2xSettings) {
```

Comment:
It used to be named something like that, but someone didn't like the "20" because they coudln't tell if it was 0.20 or 2.0. So maybe `indexCreatedBefore2x`?